.github/workflows/auto_assignment.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Auto Assign'
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+
+
+jobs:
+  add-assignees: # This needed for to create the gh issue in case of failed auto-cherry-pick
+    name: Add author to assignee
+    if: "!contains(github.event.pull_request.labels.*.name, 'Auto_Cherry_Picked')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kentaro-m/auto-assign-action@v1.2.4
```

Comment:
@omkarkhatavkar Can we add dependabot.yml for updating this GHA version for airgun?

Reply:
Done !